commands/inspect.go

```diff
@@ -126,6 +126,14 @@ func runInspect(ctx context.Context, dockerCli command.Cli, in inspectOptions) e
 						fmt.Fprintf(w, "\tKeep Bytes:\t%s\n", units.BytesSize(float64(rule.KeepBytes)))
 					}
 				}
+				if debug.IsEnabled() {
```

Comment:
Not quite sure if we should do this only when debug is enabled :thinking:

Reply:
I think if file is set, then just show it.